### PR TITLE
require rails_helper instead of spec_helper

### PR DESCRIPTION
**Pull request by naofumi-fujii**
topic
--
Generators run in RSpec 3.x will now require rails_helper instead of spec_helper. https://github.com/rspec/rspec-rails

note
--
please  ignore this PR if this gem should both work with Rspec2.x and 3.x :pray: 

fixed
--
- bundle update
- replaced spec_helper -> rails_helper

### CODE REVIEW

File: lib/rspec_kickstarter/erb_templates.rb [hunk 1]
```diff
@@ -26,7 +26,7 @@ module ERBTemplates
     BASIC_NEW_SPEC_TEMPLATE = <<SPEC
 # -*- encoding: utf-8 -*-
 
-require 'spec_helper'
+require 'rails_helper'
```
Comment (seratch) [inline lib/rspec_kickstarter/erb_templates.rb]: `BASIC_NEW_SPEC_TEMPLATE` is not a template for Rails apps. This part should not be changed.

**Comment by seratch** [comment]
Thanks, others look good to me :+1:

**Comment by seratch** [comment]
One more thing: would you please update README as well? https://github.com/seratch/rspec-kickstarter#rails-mode

**Comment by naofumi-fujii** [comment]
>update README as well?

sure :+1:

**Comment by naofumi-fujii** [comment]
i think its fixed :wrench:

**Comment by seratch** [comment]
version 1.0.2 is out. Thank you for your contribution :+1:

**Comment by naofumi-fujii** [comment]
:+1:

**Pull request merged**